alns/stopping_criteria/MaxRuntime.py

```diff
@@ -0,0 +1,41 @@
+import time
+
+from alns.State import State
+from alns.stopping_criteria.StoppingCriterion import StoppingCriterion
+
+
+class MaxRuntime(StoppingCriterion):
+    def __init__(self, max_runtime: int) -> None:
+        """
+        Criterion that stops after a maximum number of iterations.
```

Comment:
Typo/unclear: runtime, not iterations

Reply:
See [e98d90d](https://github.com/N-Wouda/ALNS/pull/64/commits/e98d90d3d08866fb3eb0f3dda14cdafc1429c265)